openedx/core/djangoapps/profile_images/tests/test_views.py

```diff
@@ -405,13 +385,79 @@ def test_remove_staff(self, mock_log):
     @patch('student.models.UserProfile.save')
     def test_remove_failure(self, user_profile_save, mock_log):
         """
-        Test that when upload validation fails, the proper HTTP response and
+        Test that when remove validation fails, the proper HTTP response and
         messages are returned.
         """
         user_profile_save.side_effect = [Exception(u"whoops"), None]
         with self.assertRaises(Exception):
-            self.client.post(self.url)
+            self.client.delete(self.url)
         self.check_images(True)  # thumbnails should remain intact.
         self.check_has_profile_image(True)
         self.assertFalse(mock_log.info.called)
         self.assert_no_events_were_emitted()
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Profile Image API is only supported in LMS')
```

Comment:
Good question. Around the time that we wrote this API, there was a push to put new functionality in openedx instead of common (to make them actual django apps and get away from the oddness of common). We did envision that the APIs could be useful in CMS, but since we had no current use case for it, we chose not to add the URLs to the CMS until a use case presented itself.

@andy-armstrong does that match your memory as well?